File: csharp/07 - Demo 3 - Refactoring/Refactoring/IntroduceParameterObject/Entry.cs
using System;

namespace Refactoring.IntroduceParameterObject
{
    public class Entry
    {
        public Entry(double value, DateTime chargeDate)
        {
            Value = value;
            ChargeDate = chargeDate;
        }

        public DateTime ChargeDate { get; }
        public double Value { get; }

    }
}

File: java/07 - Demo 3 - Refactoring/src/main/java/introduceparameterobject/Entry.java
package introduceparameterobject;

import java.util.Date;

public class Entry {
	Entry(double value, Date chargeDate) {
		this.value = value;
		this.chargeDate = chargeDate;
	}

	Date getDate() {
		return chargeDate;
	}

	double getValue() {
		return value;
	}

	private Date chargeDate;

	private double value;
}
